exercises/practice/darts/.docs/hints.md

```diff
@@ -3,9 +3,10 @@
 ## General
 
 -  This challenge is all about calculating if a point falls _on, in, or outside_ a given circle.
--  There are two different ways of calculating if a point falls on, in, or outside a circle: _Stack Overflow_ - [Equation for Testing if a Point is Inside a Circle][point-circle-equation] outlines one method.
-_DoubleRoot_ - [Position of a point relative to a circle][point-to-circle] outlines a different one.
-- This _Math is Fun_ post covers a more general [Distance Between 2 Points][distance-between-two-points] calculation.
+-  There are two different ways of calculating if a point falls on, in, or outside a circle.
+   - This _Stack Overflow_ Post: [Equation for Testing if a Point is Inside a Circle][point-circle-equation] outlines one method.
+   - This _DoubleRoot_ post [Position of a point relative to a circle][point-to-circle] outlines a different one.
+   - This _Math is Fun_ post covers a more general [Distance Between 2 Points][distance-between-two-points] calculation.
 - Because the dart board is a set of _nested_ circles, the order in which you calculate points could change the answer significantly.
```

Comment:
This is not inside the scope of change, but nested and concentric are a bit different, and these circles are indeed concentric, if we were wanting to not the nature of the nested circles.

Reply:
"concentric" feels more .... "mathy", but is indeed more accurate than "nested".  I'll change it.